Taskfile.yml

```diff
@@ -0,0 +1,334 @@
+      summary: |-
+        Install PixelatorR into an environment.
+
+        Variables:
+          DEPS: Whether to install pixelatorR dependencies. Default is false.
+
+        Example:
+        task install-pixelator-r ENV_NAME=my_env
+
+      vars:
+        DEPS: '{{ .DEPS | default "false" }}'
+      cmds:
+        - |
+          set -euo pipefail
+
+          # Determine the environment to use, handle unbound variable
+          ENV_TO_USE="{{.ENV_NAME}}"
+          if [ -z "$ENV_TO_USE" ]; then
+            ENV_TO_USE=$(basename $CONDA_DEFAULT_ENV)
+          fi
+
+          # Check if the environment exists, otherwise quit
+          micromamba env list | grep -q "$ENV_TO_USE" || {
+            echo "Environment $ENV_TO_USE does not exist. Please create it first."
+            exit 1
+          }
+
+          echo "Installing pixelatorR in environment $ENV_TO_USE"
+
+          micromamba run -n $ENV_TO_USE Rscript -e '
+            local({r <- getOption("repos")
+              r["CRAN"] <- "https://cran.r-project.org"
+              options(repos=r)
+            })
+            remotes::install_github(
+              "PixelgenTechnologies/pixelatorR",
+              upgrade="never",
+              deps={{ .DEPS | upper }}
+            )
+          '
+      silent: true
+
+  setup-env-bare:
+    desc: Create a micromamba environment with R installed
+    summary: |-
+      Create a micromamba environment with R installed
+      --------------------------------------------------------------
+
+      The environment name can be set using the ENV_NAME environment variable.
+
+      Example:
+      task setup-env ENV_NAME=my_env
+    cmds:
+      - |
+        set -euo pipefail
+
+        # Default environment name
+        ENV_NAME=${ENV_NAME:-r-pixelator}
+
+        # Check if micromamba is installed
+        if ! command -v micromamba &> /dev/null; then
+          echo "micromamba is not installed. Please install it first."
+          exit 1
+        fi
+
+        # Create the environment
+        if micromamba create -n '{{.ENV_NAME}}' -y -c conda-forge conda-forge::r-base; then
+          echo "Environment '{{.ENV_NAME}}' created successfully."
+        else
+          echo "Failed to create environment '{{.ENV_NAME}}'."
+          exit 1
+        fi
+    vars:
+      ENV_NAME: '{{.ENV_NAME | default "r-pixelator"}}'
+    silent: true
+
+  setup-env-pixelatorR:
+    desc: Create a micromamba environment with pixelatorR
+    summary: |-
+      Create a micromamba environment with pixelatorR
+      --------------------------------------------------------------
+
+      The environment name can be set using the ENV_NAME environment variable.
+
+      Examples:
+      task setup-env-pixelatorR ENV_NAME=my_env
+      task setup-env-pixelatorR ENV_NAME=my_env MICROMAMBA_ARGS="--platform osx-64"
+
+    vars:
+      MICROMAMBA_ARGS: '{{ .MICROMAMBA_ARGS | default "" }}'
+      ENV_NAME: '{{.ENV_NAME | default "r-pixelator"}}'
```

Comment:
Could be good to mention the default if you add documentation to developer docs

Reply:
Good suggestion! I will add that